Resources/MobileVLCKit/patches/0024-access_output-file-Add-error-dialog-for-write-open.patch

```diff
@@ -0,0 +1,40 @@
+From bb6baab0248c4534e25074e5e576c266f00ba057 Mon Sep 17 00:00:00 2001
+From: Soomin Lee <bubu@mikan.io>
+Date: Mon, 1 Oct 2018 15:37:57 +0200
+Subject: [PATCH 24/24] access_output: file: Add error dialog for write/open
+
+---
+ modules/access_output/file.c | 8 ++++++++
+ 1 file changed, 8 insertions(+)
+
+diff --git a/modules/access_output/file.c b/modules/access_output/file.c
+index fbefce0be8..a2a72db411 100644
+--- a/modules/access_output/file.c
++++ b/modules/access_output/file.c
+@@ -88,6 +88,9 @@ static ssize_t Write( sout_access_out_t *p_access, block_t *p_buffer )
+         {
+             if (errno == EINTR)
+                 continue;
++            if (errno == ENOSPC)
++                vlc_dialog_display_error(p_access, "record",
++                                         "error: %s", vlc_strerror_c(errno));
```

Comment:
According to Thomas there is no way to propagate Errors, which would have been the ideal solution here. This is an error that says that the user doesn't have enough space to do their recording. Since this is not something you can check in advance but is easy to fix on the user side we decided to make that one user facing

Reply:
Ok, however, in reality, the user will get a dialog stating "error: -13" without further explanation, but this is a limitation of libvlc, not of your patch.

Reply:
I wasn't aware that it's "error: -13" that's actually not okay in that case and need to be addressed